tests/hazmat/primitives/test_keywrap.py

```diff
@@ -163,6 +163,18 @@ def test_unwrap(self, backend, params):
             )
             assert params["p"] == binascii.hexlify(unwrapped_key)
 
+    @pytest.mark.parametrize(
+        "params",
+        _load_all_params("keywrap", ["kwp_botan.txt"], load_nist_vectors)
+    )
+    def test_unwrap_additional_vectors(self, backend, params):
+        wrapping_key = binascii.unhexlify(params["key"])
+        wrapped_key = binascii.unhexlify(params["output"])
```

Comment:
Let's use these vectors in both directions? Wrap the input and verify against the expected output as well